classes/class3/Homework/app.js

```diff
@@ -0,0 +1,51 @@
+var express = require('express');
+var app = express();
+var catinfo = require('./model/catinfo.js');
+var Cat = require('./model/catData').Cat;
+var exphbs = require('express-handlebars');
+var PORT = process.env.PORT || 3000;
+
+app.engine('handlebars', exphbs({defaultLayout: 'main'}));
+app.set('view engine', 'handlebars');
+app.listen(PORT, function (){
+	console.log('Running on port: ', PORT);
+});
+
+app.get('/cats/new', function (req, res){
```

Comment:
This style is NOT acceptable. You NEED to have a separate route file that you require.